src/rpc/blockchain.cpp

```diff
@@ -1207,7 +1207,7 @@ RPCHelpMan getblockchaininfo()
     UniValue obj(UniValue::VOBJ);
     obj.pushKV("chain",                 Params().NetworkIDString());
     obj.pushKV("blocks",                height);
-    obj.pushKV("headers",               pindexBestHeader ? pindexBestHeader->nHeight : -1);
+    obj.pushKV("headers",               chainman.pindexBestHeader ? chainman.pindexBestHeader->nHeight : -1);
```

Comment:
first commit: Any reason to violate clang-format on this line? I think the excessive and useless whitespace can be removed while touching this line. In line with `CalculateCurrentUsage` below.

Reply:
Done: a4d9c6fd64e1d3222e47b8aae162d472957f2d47, 65ac133803386fb19498d3c0fc5f8b27051665af